README.md

```diff
@@ -0,0 +1,6 @@
+Short instruction how to run weltyt.py in Docker:
+- Put Dockerfile and weltyt.py in one directory.
+- In Terminal enter into this directory where files are.
+- Build an image to Docker by command: [docker build -t weltyt .]
+- Run image by command: [docker run weltyt]
+- If you have problem with permission try in administration mode by command [sudo]
```

Comment:
`try to run it as root` instead of `try in administration mode by command [sudo]` (this term doesn't even exist in Linux).